controller/userController.js

```diff
@@ -82,85 +82,107 @@ const getAllusercontroller = async (req,res,next) =>{
 
 }
 
-const loginOTPController = async (req,res,next)=>{
+const sendOTP = async (req,res,next)=>{
     
-    if(!req.body.email){
-
-         res.status(400).json({message:"Please enter email "})
-    }
-    const otp = Math.floor(100000 + Math.random() * 900000).toString();
-    
-    const data = {
-        email :req.body.email,
-        otp:otp,
-        expires_at: Date.now() + 5 * 60 * 1000
-    }
-    console.log(Date.now() + 5 * 60 * 1000)
+        
+   if(!req.body.email){
+      
+      res.status(400).json({message:"No email "})
+   }
 
-   try{const result = await OTPstore(data)}
-   catch(e){return console.log(e)}
+   const otp = Math.floor(100000 + Math.random() * 900000).toString();
    
-   await sendEmail(data.email,data.otp)
-   res.status(200).json({otp:otp})
-   next()
+   let params = {
+      email: req.body.email,
+      otp: otp,
+      lastName: req.body.lastName,
+      firstName : req.body.firstName
+   }
+   
+   try {
+      const result = await userServices.sendOtpServices(params)
+      
+      res.status(200).json(result)
+   } catch (e) {
+      console.log(e)
+      res.status(500).json({message:e})
+      
+   }
+
 }
 
 const verifyOTPlogin = async (req,res,next)=>{
     
-    const {otp,email,device_type,device_token} = req.body
+   //  const {otp,email,device_type,device_token} = req.body
 
 
-   if(!otp || ! email ){
-      return res.json({message:"ENTER login and email !"})
```

Comment:
It should be an error. What you are doing it will return as a response, throw an error from the API